### PR TITLE
Update codegen dependencies

### PR DESCRIPTION
**Pull request by lblackstone**
Fix https://github.com/pulumi/kube2pulumi/issues/82

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,13 +1,186 @@
 module github.com/pulumi/kube2pulumi
 
-go 1.16
+go 1.19
```
Comment (squaremo) [inline go.mod:3]: Why this change -- are there Go 1.19 language features in use?

File: testdata/MultilineString.cs [hunk 2]
```diff
@@ -1,22 +1,21 @@
+using System.Collections.Generic;
```
Comment (squaremo) [inline testdata/MultilineString.cs:1]: Do the changes hereunder relate to those above, in this commit? I think they ought to go in 37903bb, but if they are of a kind and I'm not seeing it, it's worth explaining in the commit message how they are related.

File: go.mod (hunk 1 above)
Comment (lblackstone) [reply go.mod:3]: All of our centrally-managed GHA workflows are using 1.19 now. We have to be more careful for sdk modules, but normally track pretty close to latest for binary deps.

File: testdata/MultilineString.cs (hunk 2 above)
Comment (lblackstone) [reply testdata/MultilineString.cs:1]: No, they aren't related. I'll split them out to make it clearer.

File: go.mod (hunk 1 above)
Comment (squaremo) [reply go.mod:3]: What do you mean by "using" here? The declaration in go.mod gives the version of the language required by the module. There's generally no need for it to be consistent, or the latest, because newer releases support older versions of the language.

Comment (lblackstone) [reply go.mod:3]: I mean that the workflows use Go 1.19, so that's what we're building binaries and testing with. It also affects the `go.mod` dependencies, since the import handling has improved quite a bit since modules were introduced.

Comment (squaremo) [reply go.mod:3]: > It also affects the go.mod dependencies, since the import handling has improved quite a bit since modules were introduced.

OK, I see, using `go 1.19` changes what it puts in go.mod. (The pinning would have worked with go 1.16 too! But I shall get out the way of progress :shrug:)

**Review by squaremo — APPROVED** [review]
I messed about with trying to get a version of go-yaml that works and I'm persuaded that a replace is the way to do it here (and it still compiles).
Thanks for including the other de-bitrotting 🌟
